designer/server/src/test/scala/pl/touk/nussknacker/test/processes/WithSimplifiedScenarioActivitySpecAsserts.scala

```diff
@@ -0,0 +1,99 @@
+package pl.touk.nussknacker.test.processes
+
+import io.restassured.RestAssured.`given`
+import io.restassured.module.scala.RestAssuredSupport.AddThenToResponse
+import io.restassured.response.ValidatableResponse
+import org.scalatest.freespec.AnyFreeSpecLike
+import pl.touk.nussknacker.test.NuRestAssureMatchers
+import pl.touk.nussknacker.test.base.it.NuItTest
+import pl.touk.nussknacker.test.config.{WithSimplifiedConfigRestAssuredUsersExtensions, WithSimplifiedDesignerConfig}
+
+trait WithSimplifiedScenarioActivitySpecAsserts
+    extends AnyFreeSpecLike
+    with NuItTest
+    with WithSimplifiedDesignerConfig
+    with WithSimplifiedConfigRestAssuredUsersExtensions
+    with NuRestAssureMatchers {
+
+  implicit class VerifyCommentExists[T <: ValidatableResponse](validatableResponse: T) {
+
+    def verifyCommentExists(scenarioName: String, commentContent: String): ValidatableResponse = {
+      given()
+        .when()
+        .basicAuthAllPermUser()
+        .get(s"$nuDesignerHttpAddress/api/processes/$scenarioName/activity")
+        .Then()
+        .statusCode(200)
+        .body(
+          matchJsonWithRegexValues(
+            s"""
+               |{
+               |  "comments": [
+               |    {
+               |      "id": "${regexes.digitsRegex}",
+               |      "processVersionId": 1,
```

Comment:
Don't fix parameters. If you think that it is not necessary to check it, use regexes instead. But TBH author of the comment is quite important thing to check. I remember that there is some trick that we are forwarding the user but this user is only visible in the comment body, not as an author of the comment.
Also